app/src/main/java/cz/muni/fi/pv256/movio2/uco_396537/MainActivity.java

```diff
@@ -63,84 +85,142 @@ protected void onCreate(Bundle savedInstanceState) {
             setContentView(R.layout.activity_main);
         }
 
-        ListViewFragment listViewFragment = new ListViewFragment();
-        DetailViewFragment detailViewFragment = new DetailViewFragment();
-        listViewFragment.setAppContext(this);
+        mListViewFragment = new ListViewFragment();
+        mDetailViewFragment = new DetailViewFragment();
```

Comment:
staticky v xml, tak jak je popsáno zde https://jonassevcik.gitbooks.io/android-uvod/content/3-zivotni_cyklus.html ? To by pak ale nebylo možné v rámci jedné aktivity přepínat mezi dvěma fragmenty ne? A ve většině tutoriálu používají právě tenhle přístup, tak jsem myslel, že je ok.